### ISSUE TITLE
Display error when use tab to autocomplete

### ISSUE THREAD

**Issue by BenSYZ** [ohmyzsh/ohmyzsh#9448]
#### 1. Look for similar issues already posted (including closed ones)

There is a similar issue #7784. But it doesn't help me.

When I use tab to autocomple the command, two cases happen:
1. command start with what I have typed is unique:  
It autocomple directly. At this time everything is ok.
2. command start with what I have typed is not unique:  
Zsh will show items below to be selected. At this time, the first two character will duplicate.

For example,

```sh
➜  ~ ec<tab>
➜  ~ echo

➜  ~ echo<tab>
➜  ~ ececho
echo    echotc  echoti
```
 
#### 2. Include as much relevant information as possible

I install archlinux in virtualbox(version 6.1.16-1).

* `Linux Arch-vbox 5.9.8-arch1-1 #1 SMP PREEMPT Tue, 10 Nov 2020 22:44:11 +0000 x86_64 GNU/Linux`
* `zsh 5.8 (x86_64-pc-linux-gnu)`
* `oh-my-zsh version: 9b119866`
* `.zshrc`
```config
export ZSH="$HOME/.oh-my-zsh"
ZSH_THEME="robbyrussell"
source $ZSH/oh-my-zsh.sh
```

#### 3. Try to make sure the issue is due to Oh My Zsh

If I commit out `source $ZSH/oh-my-zsh.sh`, pure zsh works well.

I've tried xterm or alacritty, both works well.

**Comment by mcornella** [comment]
Does this help? https://github.com/ohmyzsh/ohmyzsh/wiki/FAQ#i-see-duplicate-typed-characters-after-i-complete-a-command

**Comment by BenSYZ** [comment]
I am sorry for not reading the FAQ. That helps me. Thanks a lot!

I've set `LANG=en_US.UTF-8` in `/etc/locale.conf`, but not compiled locale file. So `locale` gives me:
```
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_MESSAGES to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
...
```

After uncomment out some lines in `/etc/locale.gen` and run `locale-gen`, it works well.

ps. This problem once appeared when I installed the system(due to my careless). I ignored it at that time.
